tests/tck/features/match/MultiQueryParts.feature

```diff
@@ -26,7 +27,23 @@ Feature: Multi Query Parts
       | "Tim Duncan" | "Boris Diaw"  | "Spurs"      |
       | "Tim Duncan" | "Boris Diaw"  | "Suns"       |
       | "Tim Duncan" | "Boris Diaw"  | "Tim Duncan" |
-    When executing query:
+    # And the execution plan should be:
```

Comment:
Why comment these?

Reply:
There is something wrong in the check mechanism of operator info of tck currently.